### ISSUE TITLE
Bunch of mixed bugs (and a few minor suggestions)

### ISSUE THREAD

**Issue by JamesTripleQ** [TeamMoonstorm/Starstorm2#558]
### Bugs:
- Captain's Grandmastery skin has his hat way up in the air
![image](https://github.com/user-attachments/assets/4bb3f123-bec6-46b2-ac98-447fec87dd1c)

- In the challenges tab of the logbook:
     - `Another Bite` doesn't have a description
     - Nemesis Commando and Nemesis Mercenary have several skin unlocks, none of which can be selected in the survivor menu
     - All Nemesis Mercenary unlocks (including the survivor itself) are shown as `You have not met the prerequisites for this challenge`

### Maybe Bugs:
- Nemesis Mercenary doesn't have a survivor select sound

### Not bugs but suggestions:
- Make ScrollableLobbyUI a dependency (since it's basically needed to have the lobby not look weird)
- Make Chirr's survivor select sound slightly lower when it loops infinitely (it's annoying imo)
- ~~Add the maid skin~~

**Comment by Hevia** [comment]
Yep skin bugs are a known issue. Will be fixed soon, and no we’re never adding a maid skin hard stop. Sorry to ruin your dreams 

@nayDPz i thought we lowered the chirr character select screen sound already???

**Comment by JamesTripleQ** [comment]
yeah tbh the maid skin probably wouldn't even work well with the new model
